lib/puppet/indirector/certificate_revocation_list/rest.rb

```diff
@@ -7,4 +7,16 @@ class Puppet::SSL::CertificateRevocationList::Rest < Puppet::Indirector::REST
   use_server_setting(:ca_server)
   use_port_setting(:ca_port)
   use_srv_service(:ca)
+
+  def find(request)
+    if request.method == :find && !Puppet::FileSystem.exist?(Puppet[:hostcrl])
```

Comment:
Thanks for catching that, I originally implemented this in `#network` where this check was needed. When I shifted the implementation I didn't drop this check. It's gone now.